src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -176,9 +177,9 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_set_number(char *set_name)
 #define MPIDI_OFI_CONTEXT_BITS              MPIDI_OFI_CONTEXT_BITS_PSM2
 #define MPIDI_OFI_SOURCE_BITS               MPIDI_OFI_SOURCE_BITS_PSM2
 #define MPIDI_OFI_TAG_BITS                  MPIDI_OFI_TAG_BITS_PSM2
-/* === Compile time only === */
-#define MPIDI_OFI_MAJOR_VERSION             1
-#define MPIDI_OFI_MINOR_VERSION             5
+#define MPIDI_OFI_MAJOR_VERSION             MPIDI_OFI_MAJOR_VERSION_PSM2
+#define MPIDI_OFI_MINOR_VERSION             MPIDI_OFI_MINOR_VERSION_PSM2
```

Comment:
Seems like some pieces of the infrastructure are missing. E.g. MPIDI_OFI_MINOR_VERSION_PSM2 definition